lib/compiler/include/compiler/compiler.h

```diff
@@ -8,11 +8,13 @@ namespace FlexFlow {
 // struct SearchSolution {
 //   LabelledMultiDiGraph<PCGOperatorAttrs, ParallelTensorShape> optimized_pcg;
 //   std::unordered_map<Node, MachineView> device_assignments;
-//   /* std::unordered_map<tensor_guid_t, std::unordered_set<parallel_tensor_guid_t>> tensor_mappings; */
+//   /* std::unordered_map<tensor_guid_t,
+//   std::unordered_set<parallel_tensor_guid_t>> tensor_mappings; */
 // };
-// 
-// SearchSolution run_data_parallelize(ComputationGraph const &, MachineSpecification const &);
+//
+// SearchSolution run_data_parallelize(ComputationGraph const &,
+// MachineSpecification const &);
 
-}
+} // namespace FlexFlow
```

Comment:
You can drop the namespace comments, until there's a way to automatically insert them it's not worth the effort imo